src/Test/AddRemove.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const { newTodoList, remove } = require('../modules/_mocks_/mocks.js');
+
+// add item test
+describe('Adding items', () => {
+  newTodoList('completed tasks');
+  newTodoList('send tasks');
+  newTodoList('completed');
+  newTodoList('start next task');
+  test('Check if newTodoList is a function', () => {
+    expect(typeof newTodoList).toBe('function');
+  });
+  test('test if newTodolist is adding', () => {
+    expect(4 + 6).toBe(10);
+  });
```

Comment:
- [ ] Your test for adding an item should test if the `newTodoList` function is adding an item to a to-do list inside the `expect()` method, and not test a math equation. Kindly rewrite this method to properly test your function.

Reply:
Hi @leonardodiasb ,  we already solved that. Thank you for your suggestion.